content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -390,10 +390,10 @@ Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk,
 
 ### Mount Options
 
-A Kubernetes administrator can specify additional mount options for when a Persistent Volume is mounted on a node.
+A Kubernetes administrator can specify additional mount options for when a `PersistentVolume` is mounted on a node.
```

Comment:
Replace with just "You can". We use second person singular, to speak to the user directly, as much as possible. Do a sweep throughout the topic.